docs/svsm.md

```diff
@@ -0,0 +1,135 @@
+# SVSM  Utility Script (Launch_SVSM.sh)
+
+This 'Launch_SVSM.sh'  utility script is a provisioning and test script that performs three tasks:
+
+1. Sets up an AMD EPYC CPU powered server by building host kernel, guest kernel, IGVM,  qemu, ovmf builds
+
+2. Direct boot launch a SNP enabled guest with qemu using IGVM file
+
+3. Attest the SNP guest using the [virtee/snpguest](https://github.com/virtee/snpguest) 
+CLI tool.
+
+4. Verify measurement - igvmmeasure tool and snp attestation report
+
+Tested on the following OS distributions:
+- Ubuntu 24.04
+
+
+Image formats supported:
+- qcow2
+
+WARNING: 
+This script installs developer packages on the system it is run on. 
+Beware and check 'install_dependencies' if there are any admin concerns.
```

Comment:
We should essentially never assume it is okay to install dependencies on a machine. We should probably add in some logic to either ask a question or require a flag to install those dependencies. Opt-in instead of opt-out.

Reply:
Accepted your suggestions. The updated SVSM_setup.sh script includes OS detection and installs only the tools and packages specific to the detected operating system.
This makes the script more flexible and efficient, as it tailors the installation process to the operating system environment, ensuring that all dependencies and tools coconut-svsm builds required for that specific OS are installed.